README.md

```diff
@@ -36,6 +36,8 @@ GLOBAL OPTIONS:
    --dry-run, -d                       Conduct a dry-run: Repackage the image, but only write the Lambda layers to local disk (do not publish to Lambda)
    --description value, --desc value   The description of this layer version
    --license-info value, -l value      The layer's software license. It can be an SPDX license identifier, the URL of the license hosted on the internet, or the full text of the license
+   --compatible-runtimes value --compatible-runtimes value,
```

Comment:
Why is the option repeated?

Reply:
It is repeated because that is how the CLI would like to accept a list of values.

`./bin/local/img2lambda -i lambda-php:latest -r us-west-1 --desc "test description" -l "MIT-0" --cr nodejs6.10 --cr nodejs8.10
`

The default help does not print it in that way hence I had to manually repeat it in the README. Let me know if you would like me to remove it.

Reply:
Gotcha, makes sense, I'm still getting used the cli lib I used

Reply:
If it's not a list, and instead repeated options, then the option should be "compatible-runtime" (not plural)

Reply:
I agree. I have updated this in the latest commit